dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ContentletIndexAPIImpl.java

```diff
@@ -593,11 +599,11 @@ public void appendToBulkProcessor(final BulkProcessor bulk, final Collection<Rei
     }
 
     @Override
-    public BulkRequestBuilder appendBulkRequest(BulkRequestBuilder bulk, final ReindexEntry idx)  throws DotDataException {
-        bulk = (bulk == null) ? createBulkRequest() : bulk;
+    public BulkRequest appendBulkRequest(BulkRequest bulkRequest, final ReindexEntry idx)  throws DotDataException {
+        bulkRequest = (bulkRequest == null) ? createBulkRequest() : bulkRequest;
         Logger.debug(this, "Indexing document " + idx.getIdentToIndex());
 
-        BulkIndexWrapper bulkIndexWrapper = new BulkIndexWrapper(bulk);
+        BulkIndexWrapper bulkIndexWrapper = new BulkIndexWrapper(bulkRequest);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'bulkIndexWrapper' could be declared final](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=3712629)